electrical/readme.MD

```diff
@@ -0,0 +1,261 @@
+
+* [Techopedia Strain Relief](https://www.techopedia.com/definition/2301/strain-relief) 
+* [YouTube Wire Braid](https://www.youtube.com/watch?v=FeCs98TSsYQ)
+
+![](Pictures/Routing/route1.PNG)![](../images/electronics/route2.png)![](Pictures/Routing/route3.PNG)
+
+**Figure 6. Motor Wire Routing**
+    
+An example of how we routed our wiring is shown by Figure 6. It is important to give strain relief in the cable as you are routing it through, as well as to give extra wiring for the locations that can move. In addition, Figure 7 shows how we routed the wires through the rocker-bogie, and then from the rocker-bogie into the main body. Try to pick locations that reduce strain on the wires as the robot legs move when you are routing wires into the body.
+
+![](../images/electronics/route5.png)
+
+**Figure 7. Serial signal routing**
+
+**Table 6. Necessary Parts**
+
+Item | Ref | Qty | Image
+---- | --- | --- | ----- 
+20AWG Wire (Black) | X2 | N/A | <p align="center"> <img src="../images/components/wiring/X2.png" width="25%"> </p>
+20AWG Wire (Red) | X1 | N/A | <p align="center"> <img src="../images/components/wiring/X1.png" width="25%"> </p> 
+30AWG Wire (White) | X3 | N/A | <p align="center"> <img src="../images/components/wiring/X3.png" width="25%"> </p> 
+30AWG Wire (Blue) | X4 | N/A | <p align="center"> <img src="../images/components/wiring/X4.png" width="25%"> </p> 
+30AWG Wire (Yellow) | X5 | N/A | <p align="center"> <img src="../images/components/wiring/X5.png" width="25%"> </p> 
+30AWG Wire (Green) | X6 | N/A | <p align="center"> <img src="../images/components/wiring/X6.png" width="25%"> </p> 
+3 Pin Micro Connector | E40 | 4 | <p align="center"> <img src="../images/components/electronics/E40.png" width="25%"> </p>
+Wire Braid | E42 | N/A | <p align="center"> <img src="../images/components/electronics/E42.png" width="25%"> </p>
+Heat Shrink Tubing | E44 | 1 | <p align="center"> <img src="../images/components/electronics/E44.png" width="25%"> </p>
+
+
+The below table and figure shows an approximate amount of length of wire you will need to reach each of the motors, based on how we think is best to route the wiring through the suspension system. Cut these lengths and strip both ends of the wires.
+   
+**Table 7. Wire Lengths by Section**
+
+Section | Wires Needed | Length [inches]
+------- | ------------ | --------------- 
+A | 2x | 20AWG & 4x 30AWG | 35
+B | 2x | 20AWG & 4x 30AWG | 28
+C | 2x | 20AWG & 4x 30AWG | 48
+D | 2x | 20AWG & 3x 30AWG | 20
+E | 2x | 20AWG & 3x 30AWG | 32
+
+![Wire Lengths](../images/electronics/wire_labels.png)
```

Comment:
scale this image down a bit, use html like `<img src="/images/body/9x12_Plate_cut2.png" height="350">`

see https://github.com/nasa-jpl/open-source-rover/blob/master/examples/README_example.md

Reply:
I left these big because they are helpful when running cabling.